assets/static/js/main.js

```diff
@@ -248,8 +253,15 @@
     try {
       const { lat, lng } = getLocation()
       const response = await fetch(`/api/weather?lat=${lat}&lng=${lng}`)
+      const isCacheHit = response.headers.get('cf-cache-status') === 'HIT'
```

Comment:
What happens if this header is undefined?

Reply:
@vpetersson Considering other scenarios as a miss, be it undefined, MISS, EXPIRED, STALE etc.
https://developers.cloudflare.com/cache/about/default-cache-behavior/#cloudflare-cache-responses